datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/featureflags/GraphqlFeatureFlags.java

```diff
@@ -0,0 +1,19 @@
+package com.linkedin.datahub.graphql.featureflags;
+
+public class GraphqlFeatureFlags {
```

Comment:
We already have something called ConfigurationProvider -- why do we need this guy?